x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_item_content/boosts/boost_item_content/value_boost_form.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions } from 'kea';
+
+import {
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { RelevanceTuningLogic } from '../../../..';
+import { Boost } from '../../../../types';
+
+interface Props {
+  boost: Boost;
+  index: number;
+  name: string;
+}
+
+export const ValueBoostForm: React.FC<Props> = ({ boost, index, name }) => {
+  const { updateBoostValue, removeBoostValue, addBoostValue } = useActions(RelevanceTuningLogic);
+  const values = boost.value || [''];
+
+  return (
+    <>
+      {values.map((value, valueIndex) => (
+        <EuiFlexGroup key={valueIndex} responsive={false}>
+          <EuiFlexItem>
+            <EuiFieldText
+              value={value}
+              fullWidth
+              onChange={(e) => updateBoostValue(name, index, valueIndex, e.target.value)}
+              aria-label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.boosts.value.valueNameAriaLabel',
+                {
+                  defaultMessage: 'Value name',
+                }
+              )}
+              autoFocus
+            />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButtonIcon
+              iconType="trash"
```

Comment:
Per Davey's suggestion this morning

```suggestion
              iconType="trash"
              color="danger"
```

<img width="416" alt="" src="https://user-images.githubusercontent.com/549407/108785823-0e5bbe80-7527-11eb-88e9-1c84aee2b9fe.png">